sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -28,6 +28,10 @@ public AzureMonitorTransmitter(AzureMonitorExporterOptions options)
             options.AddPolicy(new IngestionResponsePolicy(), HttpPipelinePosition.PerCall);
 
             applicationInsightsRestClient = new ApplicationInsightsRestClient(new ClientDiagnostics(options), HttpPipelineBuilder.Build(options), host: ingestionEndpoint);
+            if (!string.IsNullOrWhiteSpace(options.ApiVersion))
```

Comment:
Why would we allow `options.ApiVersion` to be null or whitespace? Consider throw error if illegal value is provided.

Reply:
I'm expecting `options.ApiVersion` to be rarely used. Which is why I marked it EditorNonBrowseable. In this case, the null would be a valid default and the `Transmitter` would ignore it.